edmq_backend/src/data/songs.json

```diff
@@ -0,0 +1 @@
+{"index": [{"yt_title": "Seven Lions, Wooli, & Trivecta  - Island (Feat. Nevve)", "yt_id": "https://www.youtube.com/watch?v=1IFNx68qfPY", "song_title": "Island", "song_artist": "Seven Lions, Wooli & Trivecta Feat. Nevve"}, {"yt_title": "Seven Lions, SLANDER, & Dabin - First Time ft. Dylan Matthew", "yt_id": "https://www.youtube.com/watch?v=oyxklxISius", "song_title": "First Time", "song_artist": "Seven Lions, SLANDER, Dabin & Dylan Matthew"}, {"yt_title": "Seven Lions x Illenium x Said The Sky - Rush Over Me (feat. HALIENE)", "yt_id": "https://www.youtube.com/watch?v=uhA55hYnoHw", "song_title": "Rush Over Me", "song_artist": "Seven Lions & Illenium & Said The Sky Feat. HALIENE"}, {"yt_title": "GRiZ & Subtronics - Griztronics", "yt_id": "https://www.youtube.com/watch?v=K9crshqpPB0", "song_title": "Griztronics", "song_artist": "GRiZ & Subtronics"}]}
```

Comment:
should we format this?

Reply:
was this generated by the generate_song_list script?
we might not want to check this in

Reply:
Yeah I commented at the top that I probably shouldn't have checked this in. We can remove this file. I will set the parameters in the script to format the file.

Reply:
we should meet and decide on when we should collect this data and update the song json
right now it seems like we would run the script manually and check in the json file
i had an automatic system in mind